pkg/cmd/pulumi/import.go

```diff
@@ -542,14 +542,6 @@ func newImportCmd() *cobra.Command {
 				// being canceled by user request.
 				if usedProperties {
 					fmt.Print("Import failed, try specifying a different set of properties to import with.\n")
-				} else {
-					fmt.Print("Import failed, try specifying the set of properties to import with.\n")
-					if importFilePath == "" {
-						fmt.Print("This can be done by passing the property names with the --properties flag.\n")
```

Comment:
Do we think the `properties` flag is still useful then?

Reply:
Maybe if your importing lots of the same type of resource. If there are provider bugs with some fields you can work out what field set you need by importing the first resource and then tell all the rest to just use that field set.

It does occur to me we still need the diff set logic to support that correctly though, so if we want to keep --properties I'll need to add that back in.